src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -223,28 +221,32 @@
     export default defineComponent({
         name: "ModelOutput",
         beforeMount() {
-            if (!this.selectedTab) {
-                this.tabSelected(this.tabs[0]);
-            }
+            const tab = this.selectedTab ? this.selectedTab : this.tabs[0]
+            this.tabSelected(tab);
         },
         data: () => {
-            const tabs: (keyof Translations)[] = ["map", "bar"];
+            const tabs: (keyof Translations)[] = [ModelOutputTabs.Bar];
 
-            if (!inactiveFeatures.includes("BubblePlot")) {
-                tabs.push("bubble");
+            if (switches.tableTab) {
+                tabs.push(ModelOutputTabs.Table);
             }
-            tabs.push("comparison");
 
-            if (switches.tableTab) {
-                tabs.push("table")
+            tabs.push(ModelOutputTabs.Map);
+            tabs.push(ModelOutputTabs.Comparison);
+
+            if (!inactiveFeatures.includes("BubblePlot")) {
+                tabs.push(ModelOutputTabs.Bubble);
```

Comment:
Updated the order whilst I was here as something Rachel has wanted to change for a while